packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.service.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```

Comment:
**nitpick:**  Please remove the disabling of lint rules.

Reply:
Done.